README.rst

```diff
@@ -70,9 +70,110 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
+
+     ::
+
+       $ git clone https://github.com/coala/coala-html.git
+
+   - Download and install the latest version of Node.js from `official website <https://nodejs.org/en/download/>`__.
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip install coala-bears
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ npm install -g bower
+
+
+   - Run **coala-html** to display results from **coala-json**
+
+     ::
+
+       $ coala-html --dir ..\Tests
+
+
+2. Installation instructions for **Ubuntu/Linux** users:
+
+   - Clone the **coala-html** repository
+
+     ::
+
+      $ git clone https://github.com/coala/coala-html.git
+
+   - Install latest version of **Node.js**
+
+     a. For **Ubuntu** users:
+
+        * *Configure Node.js PPA*
+
+          ::
+
+           $ sudo apt-get install curl python-software-properties
+           $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+        * *Install Node.js and npm*
+
+          ::
+
+           $ sudo apt-get install nodejs
+
+     b. For installing Node.js via Package manager on other distributions, refer `official guide <https://nodejs.org/en/download/package-manager/>`__.
+
+   - *Check Node.js and npm version*
+
+     You can find more details about current version on node.js `official website <https://nodejs.org/>`__.
+
+     ::
+
+       $ node -v
+
+       $ npm -v
+
+   - Install **bower** to build static web pages
+
+     ::
+
+       $ sudo npm install -g bower
+
+   - Change the directory to **coala-html** and run following command to install dependencies
+
+     ::
+
+       $ python3 setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip3 install coala-bears
+
+   - Allow permission to **root user** to install packages required by bower
```

Comment:
One final change. This actually gives permission from **root** back to the **user**, so maybe change this to, `Change permissions to install packages required by bower`

Reply:
On a more technical note, `ownership` should be used here instead of `permissions`, since actual ownership(owner and group) is changed whereas the permissions remain the same.